common/src/main/java/com/viaversion/viaversion/rewriter/ItemRewriter.java

```diff
@@ -234,8 +234,45 @@ public void registerCooldown(C packetType) {
         });
     }
 
-    // 1.14.4+
+
     public void registerMerchantOffers(C packetType) {
+        protocol.registerClientbound(packetType, new PacketHandlers() {
+            @Override
+            protected void register() {
+                map(Types.STRING); // 0 - Channel
+                handlerSoftFail(wrapper -> {
+                    final String channel = wrapper.get(Types.STRING, 0);
+                    if (channel.equals("MC|TrList")) {
+                        merchantOffersRewriter().handle(wrapper);
+                    }
+                });
+            }
+        });
+    }
+
+    public PacketHandler merchantOffersRewriter() {
```

Comment:
Just make this a void method since the handler is never used directly

Reply:
Done